src/pages/academy/gameSimulator/subcomponents/chapterPublisher/ChapterPublisherEditor.tsx

```diff
@@ -81,74 +71,79 @@ const ChapterEditor = React.memo(({ chapterDetail, checkpointFilenames }: Chapte
   const saveChapter = async () => {
     const updatedChapter = {
       openAt: openDate.toISOString(),
-      closeAt: inAYear(openDate).toISOString(),
+      closeAt: dateOneYearFromNow(openDate).toISOString(),
       title,
       filenames: chosenFiles,
       imageUrl,
       isPublished
     };
 
     const confirm = window.confirm(
-      `Are you sure you want to save changes to ${JSON.stringify(updatedChapter)}`
+      `Are you sure you want to save changes to Chapter ${id}: ${title}?\n\nChapter Information: ${JSON.stringify(
+        updatedChapter
+      )}`
     );
     if (!confirm) {
       return;
     }
     const response =
-      parseInt(id) === createChapterIndex
+      parseInt(id) === defaultChapter.id
         ? await updateChapterRequest('', { story: updatedChapter })
         : await updateChapterRequest(id, { story: updatedChapter });
 
     alert(response);
   };
 
   const deleteChapter = async () => {
-    const confirm = window.confirm('Are you sure you want to delete this chapter?');
+    const confirm = window.confirm(`Are you sure you want to delete Chapter ${id}: ${title}?`);
     if (confirm) {
       const response = await deleteChapterRequest(id);
       alert(response);
     }
   };
 
   const clearChanges = () => {
-    const confirm = window.confirm('Are you you want to clear changes for this chapter?');
+    const confirm = window.confirm(
+      `Are you sure you want to clear changes for Chapter ${id}: ${title}?`
+    );
     if (confirm) {
       setRender(!rerender);
       alert('Cleared changes');
     }
   };
 
-  const simulateChapter = async () => {
-    SourceAcademyGame.getInstance().setChapterSimStack(chosenFiles);
-    await callGameManagerForSim();
-  };
-
   return (
     <>
       <h4>
-        Title: <input className="bp4-input" type="text" {...titleProps} />
+        Title: <input className="bp4-input" type="text" placeholder="New title" {...titleProps} />
       </h4>
       <b>Open date: </b>
       {openDate && getStandardDateTime(openDate.toISOString())}
       <DatePicker
-        onChange={(date: Date) => {
+        onChange={(date: Date | null) => {
           date && setOpenDate(date);
         }}
+        showActionsBar={true}
+        highlightCurrentDay={true}
```

Comment:
Nit: Unnecessary `={true}`

Reply:
Addressed in [664d352](https://github.com/source-academy/frontend/pull/2836/commits/664d3528c4fc83f110bb07756f944cd7778ae46e).